File: taco.js
function tacofy(word) {
    let letterDict = {'a':'beef', 'e':'beef', 'i':'beef', 'o': 'beef', 'u':'beef', 't':'tomato', 'l':'lettuce', 'c':'cheese', 'g':'guacamole', 's':'salsa'};
    let wordArr = word.toLowerCase().split('')
    let newArr = wordArr.map((x)=> letterDict[x] || '');
    newArr.unshift('shell');
    newArr.push('shell');
    let fixedArr = newArr.filter((x) => x !== '');
   return fixedArr;
  }

File: taco.py
def tacofy(word):  
    word = word.lower()
    end = ['shell']
    front = ['shell']
    beefs = ['a', 'e', 'i', 'o', 'u']
    ingreds = []
    for x in range(len(word)):
        if word[x] in beefs:
            ingreds.append('beef')
        elif word[x] == 't':
            ingreds.append('tomato')
        elif word[x] == 'l':
            ingreds.append('lettuce')
        elif word[x] == 'c':
            ingreds.append('cheese')
        elif word[x]== 'g':
            ingreds.append('guacamole')
        elif word[x] == 's':
            ingreds.append('salsa')
    return front + ingreds + end